addons/mail/models/mail_activity.py

```diff
@@ -667,7 +667,11 @@ def activity_schedule(self, act_type_xmlid='', date_deadline=None, summary='', n
         if isinstance(date_deadline, datetime):
             _logger.warning("Scheduled deadline should be a date (got %s)", date_deadline)
         if act_type_xmlid:
-            activity_type = self.sudo().env.ref(act_type_xmlid)
+            try:
+                activity_type = self.sudo().env.ref(act_type_xmlid)
```

Comment:
I think it's cleaner to use `raise_if_not_found=False`

Reply:
Nice. thanks! Done :+1: